packages/rancher-logging/overlay/templates/loggings/rke/daemonset.yaml

```diff
@@ -0,0 +1,52 @@
+{{- if .Values.additionalLoggingSources.rke.enabled }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: "{{ .Release.Name }}-rke-aggregator"
+  namespace: "{{ .Release.Namespace }}"
+spec:
+  selector:
+    matchLabels:
+      name: {{ .Release.Name }}-rke-aggregator
+  template:
+    metadata:
+      name: "{{ .Release.Name }}-rke-aggregator"
+      namespace: "{{ .Release.Namespace }}"
+      labels:
+        name: {{ .Release.Name }}-rke-aggregator
+    spec:
+      containers:
+        - name: fluentd
+          image: "{{ template "system_default_registry" . }}{{ .Values.images.fluentbit.repository }}:{{ .Values.images.fluentbit.tag }}"
+          volumeMounts:
+            - mountPath: /var/lib/rancher/rke/log/
+              name: indir
+            - mountPath: /var/lib/rancher/logging/
+              name: outdir
+            - mountPath: /var/lib/docker/containers/
+              name: containers
+            - mountPath: /fluent-bit/etc/
+              name: config
+      volumes:
+        - name: indir
+          hostPath:
+            path: /var/lib/rancher/rke/log/
+        - name: outdir
+          hostPath:
+            path: /var/lib/rancher/logging/
+        - name: containers
+          hostPath:
+            path: /var/lib/docker/containers/
+        - name: config
+          configMap:
+            name: "{{ .Release.Name }}-rke"
+      {{- $total_tolerations := concat (.Values.tolerations) (.Values.fluentbit_tolerations) }}
```

Comment:
Do we include fluentbit tolerations here if its a fluentd ?

Reply:
The ds is a fluentbit

Reply:
Oh, I was looking at line 19, that still says fluentD, the image is using fluentbit.

Reply:
opps will fix